src/bin/facilitator-init.ts

```diff
@@ -93,17 +96,29 @@ commander
     facilitatorConfig.database.path = dbPath;
     facilitatorConfig.originChain = originChainId;
     facilitatorConfig.auxChainId = options.chainId;
-    const setFacilitator = (chainId: string, rpc: string, password: string): void => {
+    const setFacilitator = (chainid: string, rpc: string, password: string): void => {
       const account: Account = Account.create(new Web3(null), password);
 
-      facilitatorConfig.chains[chainId] = new Chain(rpc, account.address);
+      facilitatorConfig.chains[chainid] = new Chain(rpc, account.address);
+      // const envVariableNameForWorkerPassword = `${ENV_WORKER_PASSWORD_PREFIX}${account.address}`;
+      // process.env[envVariableNameForWorkerPassword] = password;
 
       facilitatorConfig.encryptedAccounts[account.address] = account.encryptedKeyStore;
     };
 
     setFacilitator(originChainId, options.originRpc, options.originPassword);
     setFacilitator(options.chainId, options.auxiliaryRpc, options.auxiliaryPassword);
 
+    // const config = new Config(mosaicConfig, facilitatorConfig);
+    // const repositories = await Repositories.create(config.facilitator.database.path);
+    // const seedData = new SeedData(
+    //   config,
+    //   repositories.gatewayRepository,
+    //   repositories.auxiliaryChainRepository,
+    //   repositories.contractEntityRepository,
+    // );
+    // await seedData.populateDb();
```

Comment:
Please check if this code should be commented? When is seed data populated?
If yes, then please delete it.

Reply:
We can not this open up till we have dev chains and our integration test don't start using it. Keeping it here so that we can just uncomment it later and can also test now by manually uncommenting and running it.

Reply:
Will also write a note in the ticket we have for integration tests to open this up.